### PR TITLE
update the global variables using scss

### PR DESCRIPTION
**Pull request by xzci**
- add color variables
- add font variables
- add text variables

### Descriptions

- Link to UI design (if possible):

### Provide screenshots of no warning, or error in the console.

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Review by tatoMa — COMMENTED** [review]
Looks good but we also need font files or CSS font import to use the fonts you set up on this global scss file, right?

Also, try to modify a line of text on the index page with any header style you have created as an example for others to use.

**Review by HarryuCATGREY — APPROVED** [review]
Should all css file write in one file?

**Comment by xzci** [comment]
image
image

File: pages/index.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Head from "next/head";
 import Image from "next/image";
 import { Inter } from "@next/font/google";
+
 import styles from "@/styles/Home.module.css";
 
 const inter = Inter({ subsets: ["latin"] });
```
Comment (tatoMa) [inline pages/index.tsx:7]: Take a look at the font section on Nextjs doc, you will find what is this line for

**Review by tatoMa — COMMENTED** [review]
Why keep 2 same copy of global styles in both globals.scss and global.css files?

**Review by tatoMa — COMMENTED** [review]
Also, take a look at what is _app file does on the Nextjs doc.

**Comment by xzci** [comment]
image
update the font variables
- can use font a new class name which is the same as the documents
- please use 'npm i sass' to add the support of sass

**Comment by Cameron933** [comment]
plz update & merge main branch

**Comment by tatoMa** [comment]
Is it only me? I have this error when I try to open a dev server.
image
image

**Comment by xzci** [comment]
Without this module Nextjs does not allow to use scss/sass directly(must compile to css first)please use 'npm i sass' to add the support of sass

**Comment by tatoMa** [comment]
> 

So my question is, why not provide a modified package.json with this sass package installed? So that everyone does not need to install this package individually.

**Comment by xzci** [comment]
I'm worried that some of the modules I learned and tried earlier might contaminate the current PACKAGE.JSON files.

**Comment by tatoMa** [comment]
> 

I don't get it. So you want to let everyone install this dependency individually rather than globally install it once? 

You have something using the sass package for sure. Using this sass function without the related library provided is nonsense.

**Comment by xzci** [comment]
I will mark that, and there is a conflicting issue when I pull the JSON file, please help me to fix it.

File: package.json [hunk 2]
```diff
@@ -17,15 +17,16 @@
     "yarn": "please-use-only-npm"
   },
   "dependencies": {
-    "@next/font": "13.1.6",
+    "@next/font": "^13.1.6",
```
Comment (tatoMa) [inline package.json:20]: Did this line update by you installed the sass library or have you modified it?

**Review by tatoMa — COMMENTED** [review]
image
It is not a good location for a font file. The pages folder only contains page files. It is automatically handled by Next router.

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**